site/docs/v1/tech/apis/_two-factor-start-response-body.adoc

```diff
@@ -0,0 +1,59 @@
+==== Response Body
+
+[.api]
+[field]#code# [type]#[String]#::
+A valid Multi-Factor verification code. 
+
+[field]#methods# [type]#[Array]#::
+A list of configured authentication methods. This field will be omitted if none have been configured.
+
+[field]#methods``[x]``.id# [type]#[String]#::
+The identifier for this method.
+
+[field]#methods``[x]``.lastUsed# [type]#[Boolean]#::
+`true` if this method was used most recently.
+
+[field]#methods``[x]``.method# [type]#[String]#::
+The type of this method. There will also be an object with the same value containing additional information about this method.  The possible values are:
++
+* `authenticator`
+* `email`
+* `sms`
+
+[field]#methods``[x]``.authenticator# [type]#[Object]#::
+An object with additional configuration for TOTP authentication methods. Only present if [field]#methods``[x]``.method# is `authenticator`. 
+
+[field]#methods``[x]``.authenticator.algorithm# [type]#[String]#::
+The algorithm used by the TOTP authenticator. The possible values are:
++
+* `HmacSHA1`
+* `HmacSHA256`
+* `HmacSHA512`
```

Comment:
These are all valid values, currently we will only ever return `HmacSHA1`.

Reply:
Great, will note that and make it clear.